src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -111,6 +111,7 @@ def load_arguments(self, _):
         help='Space-separated list of availability zones into which to provision the resource.',
         choices=['1', '2', '3']
     )
+    edge_zone = CLIArgumentType(help='The name of edge zone.', is_preview=True, min_api='2020-08-01')
```

Comment:
You could use this type for edge zone and add min_api version for your parameter
https://github.com/Azure/azure-cli/blob/3f3aafdd3fec06d66d318eeac7cbc0e70f124107/src/azure-cli-core/azure/cli/core/commands/parameters.py#L251